tests/database/test_database.py

```diff
@@ -42,6 +43,25 @@ def test_create_agent_profile() -> None:
     AgentProfile.delete(pk)
 
 
+def test_create_custom_dimension() -> None:
+    custom_dimension = CustomEvaluationDimension(
+        name="verbosity_custom",
+        description="The verbosity of the conversation",
+        range_low=0,
+        range_high=10,
+    )
+    custom_dimension.save()
+    pk = custom_dimension.pk
+    dimension = CustomEvaluationDimension(uuid_str=pk)
```

Comment:
`CustomEvaluationDimension(uuid_str=pk)` this is not how you fetch data from the Redis database